gloo-edge/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+GLOO_NS=gloo-system
+TRAEFIK_NS=kube-system
+
+# Uninstall default Trafeik resources as gloo-edge and traefik 
+# are mutally exclusive
+
+kubectl config set-context --current --namespace=$TRAEFIK_NS
+helm uninstall traefik
```

Comment:
we should not uninstall traefik, we are not doing this for any other app. This will be made optional in future but we should not uninstall traefik.

Reply:
but with `Traefik` this will not install as there is a port clash. Users will then need to remove the app when they create cluster before they choose. IMHO these kind of scenarios are pretty common when you have multiple similar apps in marketplace

Reply:
Yes, Right now apps are using the ingress they are compatible with but we should not uninstall something already present on the cluster.

Reply:
Then I feel this has to wait until the `optional` Traefik  feature is added. As both these mutualy exclusive

Reply:
you can add it without the part where you remove traefik as users can create cluster without traefik installed

Reply:
You can today create clusters without traefik, you have to deselect from the UI and even from the CLI you can do that

Reply:
ok I have removed the uninstall from my script. But ideally it will be nice to have  way to say mutually exclusive apps. More for a developer convinience.